---
title: "Quick and Dirty Checklist for Careful and Clean aDNA Study Review"
date: 2017-06-10
permalink: /posts/2017/06/quick-dirty-adna-checklist.md/
tags:
  - adna
  - review
  - checklist
---

With ever decreasing sequencing costs, and easier and high-yielding methods, the number of ancient DNA studies being published are rapidly increasing. Thus, the amount of palaeo- and archaeogenetic papers requiring peer review are also increasing. However, the ancient DNA community remains small compared to other areas of biology.
<!-- here put graphs reflecting increase in ancient dna citations/studies -->
<!-- https://academia.stackexchange.com/questions/3439/easy-way-to-see-historical-trend-in-keywords-in-publications -->

To assist non-ancient DNA specialists in peer-reviewing an ancient DNA study, I've written a checklist of questions to consider when assessing the authenticity of a study.

1. Has the sample been directly dated? Does the sample come from a stratigraphically secure layer?
2. Has the sample been excavated from environments suitable for ancient DNA preservation
  * Although methods are improving, warm and wet will generally less well preserved ancient DNA
2. Were the libraries generated ancient DNA clean room?
  * What precautions (clean room clothing, UVing, clean reagents)?
  * Sample decontamination procedures?
  * Were samples indexed in the clean room prior sequencing
3. Are the extraction and library controls clean?
4. Does the DNA actually look ancient?
  * Damage patterns - mapDamage, good reference, UDG treatment
  * Fragmentation - should be short, should not have to shear!
4. Does the result make biological sense
  * If microbiome - does the composition match modern taxonomic profiles?
  * If dietary DNA - is the species expected at that time (Maize shouldn't be in European Neanderthals)
  * If phylogeny - is sample branch shorter than modern sister specimens? Does the approximate date make sense? Does the specimen fall in the expected clade?
  * Is the genome 'clean'? E.g. a haploid bacterial genome should only have homozygous positions
  * If human - has a contamination test been perform (e.g. contaMix/Schmutzi)

Read complexity, deduplication, database bias (eukaroytic)

<!-- Include citations - OA where possible -->

If you have any queries, comments or additional suggestions to the list please feel free to [email me](jfy133@gmail.com) or contact me on [twitter](www.twitter.com/jafellowsyates)
